cmd/backfill-index/main.go

```diff
@@ -161,7 +188,12 @@ func main() {
 		log.Fatalf("creating index client: %v", err)
 	}
 
-	rekorClient, err := client.GetRekorClient(*rekorAddress, client.WithNoDisableKeepalive(true), client.WithRetryCount(10))
+	opts := []client.Option{client.WithNoDisableKeepalives(!*rekorDisableKeepalives)}
```

Comment:
Bob had mentioned last time, but the double negatives are a bit confusing, especially since we negate the input. Do you think it would be cleaner to switch these to `DisableKeepAlive`?

Reply:
I tried that on the first iteration of this PR, the problem is in `GetRekorClient`, if we had this:

```
defaultTransport.DisableKeepAlives = o.DisableKeepalives
```
as opposed to [conditioning on the negative](https://github.com/sigstore/rekor/pull/2146/files#diff-5b4dad70b1086e28145daaaef5434d221386a6cadd8dcae4f01a9e0eaa8cceaeR41), then if the option is unset it actually reverses the default behavior and several unit tests fail. The double negative helps distinguish whether the setting was unset or explicitly false.